components/project-card/project-card-simple.jsx

```diff
@@ -196,9 +199,13 @@ class ProjectCard extends React.Component {
                 creatorClickHandler={(event, name) =>
                   this.handleCreatorClick(event, name)
                 }
+                className="my-1 body-small"
               />
               {this.props.onModerationMode && (
-                <Description description={this.props.description} />
+                <Description 
+                  description={this.props.description}
+                  className="mt-3"
```

Comment:
The two `<Description>` use cases both have `mt-3` as classname so we might as well remove this props and just move `mt-3` as default class in `description.jsx`